config/main.py

```diff
@@ -823,6 +840,9 @@ def startup(ctx):
     config_db = ctx.obj['config_db']
     interface_name = ctx.obj['interface_name']
 
+    if interface_name_is_valid(interface_name) is False:
+        ctx.fail("Enter valid interface name!!")
```

Comment:
I think this message could be less shouty and more descriptive. Maybe something like, 

` ctx.fail("Error: '{}' is not a valid interface name.".format(interface_name))`

Reply:
Addressed the comments. Please review.